src/main/java/rx/schedulers/EventLoopsScheduler.java

```diff
@@ -27,7 +27,7 @@
 import java.util.concurrent.ThreadFactory;
 import java.util.concurrent.TimeUnit;
 
-/* package */class EventLoopsScheduler extends Scheduler {
+public class EventLoopsScheduler extends Scheduler {
```

Comment:
But since EventLoopsScheduler was package private, it could be moved into ```rx.internal.schedulers``` and be hidden from the public docs.